hub/experimental/convert_to_hub3.py

```diff
@@ -32,7 +32,7 @@ def dataset_to_hub3(hub2_dataset):
                 aws_session_token=provider.aws_session_token,
                 region_name=provider.aws_region,
                 endpoint_url=provider.endpoint_url,
-                expiration=provider.expiration,
+                expiration=str(provider.expiration),
```

Comment:
let's add tests for both expiration and slice

Reply:
for slice there is a test enabled